mern/server/DAO/QA_dao.js

```diff
@@ -0,0 +1,60 @@
+import QA from "../models/question_answer.js";
+
+//Class of DAO functions related to the QA entities
+export default class QA_DAO {
+  //Empty constructor
+  constructor(props) {
+    super(props);
+  }
+
+  //Gets the QA by ID
+  async getQAByID(req, res) {
+    try {
+      const qa = await QA.findOne({ id: req.body.id });
+      res.status(200, { qa });
+    } catch (e) {
+      res.status(400, "No Qa Exists with Given Session ID");
+    }
+  }
+
+  //Creates a new QA
+  async createQA(req, res) {
+    try {
+      const qa = await QA.create({
+        id: req.body.id,
+        question: req.body.question,
+        intent: req.body.intent,
+        question_included: req.body.question_included,
+      });
+      res.status(200, { id: qa.id });
```

Comment:
Incorrect way of setting res.status.

It should be res.status(200).json(qa.id)

Reply:
👀 The docs I read lied to me